packages/rum-core/test/common/fetch-patch.spec.js

```diff
@@ -117,7 +117,7 @@ describe('fetchPatch', function() {
     })
     it('should reset fetchInProgress global state', function(done) {
       expect(globalState.fetchInProgress).toBe(false)
-      window.fetch('http://localhost:54321/').then(done, done)
+      window.fetch('http://localhost:54321/').then(done, () => done())
```

Comment:
to avoid jasmine printing deprecation warnings. done(err) was causing issues.

Reply:
shouldn't it be `.then(done, fail)` instead.

Reply:
Not really,  that url never exists so its an error